velox/docs/functions/spark/string.rst

```diff
@@ -283,6 +283,26 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring_index('aaaaa', 'aa', 5); -- "aaaaa"
         SELECT substring_index('aaaaa', 'aa', -5); -- "aaaaa"
 
+.. spark:function:: toprettystring(x) -> varchar
+
+    Returns pretty string for all scalar type except Timestamp. It has several differences with casting value to string:
+
+    - It prints null values (either from column or struct field) as "NULL".
+
+    - It prints binary values (either from column or struct field) using the hex format. ::
+
+        SELECT toprettystring('spark');  -- "spark"
+        SELECT toprettystring(12);  -- "12"
+        SELECT toprettystring(null);  -- "NULL"
+        SELECT toprettystring(cast('abcd' as binary));  -- "[61 62 63 64 65 66]"
+
+.. spark:function:: toprettystring(x, timeZone) -> varchar
```

Comment:
I think `timeZone` can be removed and let the above doc for `toprettystring(x)` cover timestamp type. TimeZone is actually read from Velox config. And like other time/date function, Gluten will not directly pass it to native function.

Reply:
Thanks! I change the implement but forget the document.